test/hotspot/jtreg/serviceability/sa/TestG1HeapRegion.java

```diff
@@ -59,11 +59,11 @@ private static void checkHeapRegion(String pid) throws Exception {
         try {
             agent.attach(Integer.parseInt(pid));
             G1CollectedHeap heap = (G1CollectedHeap)VM.getVM().getUniverse().heap();
-            HeapRegion hr = heap.hrm().heapRegionIterator().next();
-            HeapRegion hrTop = heap.hrm().getByAddress(hr.top());
+            G1HeapRegion hr = heap.hrm().heapRegionIterator().next();
```

Comment:
"g1HeapRegionIterator"?

Reply:
And here also it seems you agreed with this suggestion but the change was never made.

Reply:
As mentioned earlier, I also think the SA changes should be done here in this CR since they are fairly minor and related to single methods, so not as encapsulated as changes to the remaining class names (for which I already created an RFR https://bugs.openjdk.org/browse/JDK-8331385).